k8s/object.bzl

```diff
@@ -123,7 +123,7 @@ def _impl(ctx):
     return struct(runfiles = ctx.runfiles(files = [
         ctx.executable.resolver,
         ctx.outputs.substituted,
-    ] + list(ctx.attr.resolver.default_runfiles.files) + all_inputs))
+    ] + list(ctx.attr.resolver.default_runfiles.files.to_list()) + all_inputs))
```

Comment:
It won't be needed after https://github.com/bazelbuild/rules_k8s/pull/300/files is in because transitive_files take a depset.

Reply:
merged and fixed